transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -877,6 +896,7 @@ public void run() {
 
     @Override
     public final ChannelPipeline fireChannelActive() {
+        autoFlush = channel.config().isAutoFlush(); // TODO: Have callback when changed.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)